CP1404_practicals/prac_05/emails.py

```diff
@@ -9,7 +9,7 @@ def main():
     while user_email != "":
         name = get_name_from_email(user_email)
         confirm_name = input(" Is your name {}? (Y/N): ".format(name)).upper()
```

Comment:
The variable name 'confirm_name' may be able to be improved by making it sound more like a noun, rather than a verb (e.g. confirmation). You can test this by checking if your variable names fit into the sentence: "This variable stores...<variable name>".